src/serve/authorization.test.ts

```diff
@@ -90,11 +90,6 @@ describe("disabled workflow context", () => {
       });
       expect(called).toBeTrue();
     });
-
-    test("shouldn't throw on cancel", () => {
-      const dontThrow = disabledContext.cancel;
-      expect(dontThrow).not.toThrowError();
-    });
```

Comment:
can we add tests for what needs to happen for cancel and fail? We can check the error type of message